apps/condo/domains/property/components/BasePropertyForm/index.tsx

```diff
@@ -67,6 +66,19 @@ const BasePropertyForm: React.FC<IPropertyFormProps> = (props) => {
         return formData
     }, [initialValues])
 
+    const { combiner, requiredValidator } = useValidations()
+    const addressValidator = {
```

Comment:
Why didnt we add this validator in useValidations?

Reply:
since it's page-related and uses states inside